lib/movie.dart

```diff
@@ -0,0 +1,30 @@
+class Movie {
+  final String title;
+  final String originalTitle;
+  final String releaseDate;
+  final String voteAverage;
+  final String genres;
+  final String overview;
+  final String backdrop;
+
+  Movie(
+      this.title,
+      this.originalTitle,
+      this.releaseDate,
+      this.voteAverage,
+      this.genres,
+      this.overview,
+      this.backdrop,
+      );
```

Comment:
It'd be better to have this as a named constructor, specially when you have several arguments of the same type, to avoid mistakes later.

Reply:
I named it "Movie.fromString", assuming that eventually I'll have a Movie.fromJson or something similar. Maybe I'll have to rename it again when that time comes, because I'm not sure I got it right.

Reply:
Hmm, didn't mean that tbh! I meant having a constructor like:
```
  Movie({
      required this.title,
      required this.originalTitle,
      required this.releaseDate,
      required this.voteAverage,
      required this.genres,
      required this.overview,
      required this.backdrop,
});
```
To be able to call the constructor as this:
```
final movie = Movie(
  title: 'Super Mario',
  releaseDate: 'DD-MM-YYYY',
  ...
);
```
Instead of having positional arguments, thus reducing typing errors!

Reply:
We'll handle the constructor for the JSON in a later time :)